src/bin/pg_autoctl/cli_do_tmux.c

```diff
@@ -649,13 +665,74 @@ tmux_stop_pg_autoctl(TmuxOptions *options)
 			break;
 		}
 
+		/* sleep enough time that the processes might already be dead */
 		sleep(1);
 	}
 
 	return success;
 }
 
 
+/*
+ * tmux_has_session runs the command `tmux has-session -f sessionName`.
+ */
+static bool
+tmux_has_session(const char *sessionName)
+{
+	Program program;
+	int returnCode;
+
+	char tmux[MAXPGPATH] = { 0 };
+	char command[BUFSIZE] = { 0 };
+
+	if (!search_path_first("tmux", tmux))
+	{
+		log_fatal("Failed to find program tmux in PATH");
+		return false;
```

Comment:
I don't think this will ever happen, but we won't handle this as an error when calling `tmux_has_session`. 
The reason is that the return value of this function is used as a way to signal if the session exists, not if an error happens. Is this on purpose or an oversight?

Reply:
Fixed, that was an oversight indeed.